Lib/fontTools/otlLib/builder.py

```diff
@@ -589,16 +587,15 @@ def build(self):
                 if builder is not None:
                     builder.addSubtableBreak()
                 continue
-            val1, valFormat1 = makeOpenTypeValueRecord(
-                value1, pairPosContext=True)
-            val2, valFormat2 = makeOpenTypeValueRecord(
-                value2, pairPosContext=True)
+            valFormat1, valFormat2 = 0, 0
+            if value1: valFormat1 = value1.getFormat()
```

Comment:
i don't like if statements in one line, even if very short. I prefer consistency.